nixos/modules/flyingcircus/tests/default.nix

```diff
@@ -11,26 +11,25 @@
 
   mongodb = hydraJob (import ./mongodb { inherit system; }) ;
 
-  mysql_5_5 = hydraJob
-    (import ./percona.nix {
-      inherit system;
-      percona = pkgs.mysql55;
-    });
-
   nodejs6 = hydraJob
     (import ./nodejs6.nix {
       inherit system;
     });
 
-  percona_5_7 = hydraJob
-    (import ./percona.nix {
+   mysql_5_5 = hydraJob
+    (import ./mysql.nix {
+      inherit system;
+      mysql55 = true;
+    });
+  mysql_5_6 = hydraJob
+    (import ./mysql.nix {
       inherit system;
-      percona = pkgs.percona56;
+      mysql56 = true;
     });
-  percona_5_6 = hydraJob
-    (import ./percona.nix {
+  mysql_5_7 = hydraJob
+    (import ./mysql.nix {
       inherit system;
-      percona = pkgs.percona57;
+      mysql57 = true;
     });
 
```

Comment:
We have a better pattern for this: inherit(import ...) test1 test2 test3.

I'll update the code myself to show the pattern.